paasta_tools/cli/cmds/status.py

```diff
@@ -2241,6 +2253,20 @@ def status_marathon_job_human(
         )
 
 
+def _get_paasta_status_version(args, system_paasta_config) -> bool:
+    if args.new:
+        return True
+    elif args.old:
+        return False
+    else:
+        if system_paasta_config.get_paasta_status_version() == "old":
+            return False
+        elif system_paasta_config.get_paasta_status_version() == "new":
+            return True
+        else:
+            return True
```

Comment:
nit: you can simplify this to `system_paasta_config.get_paasta_status_version() == "new"` maybe?  I guess that doesn't cover the case where it's neither "new" nor "old" but I don't think we're really going to encounter that (since we control it anyway)

Reply:
Is like that exactly to cover the case where it's neither "new" nor "old" since I don't trust humans and they can have typos. The other thing is that all this code is probably gonna get deleted soon once we wipe all the `--old` code